bundler/Rakefile

```diff
@@ -139,29 +139,39 @@ end
 
 load "../util/automatiek.rake"
 
-# We currently ship Molinillo master branch as of
-# https://github.com/CocoaPods/Molinillo/commit/b4a510349e6893486b9eb44a71734c05a8034256
-# plus the following patches:
-# * Require `tsort` relatively to use our vendored version.
-# * Allow full customization of Molinillo's conflict error message
-#   (https://github.com/CocoaPods/Molinillo/pull/164).
-desc "Vendor a specific version of molinillo"
-Automatiek::RakeTask.new("molinillo") do |lib|
-  lib.version = "master"
-  lib.download = { :github => "https://github.com/CocoaPods/Molinillo" }
-  lib.namespace = "Molinillo"
+# We currently ship a custom version of pub_grub that includes the following
+# patches:
+#
+# * https://github.com/jhawthorn/pub_grub/pull/15, for easier handling and
+#   customization of the "root package", which we call "Gemfile" in our error
+#   messages.
+#
+# * https://github.com/jhawthorn/pub_grub/pull/16, for easier access to pub_grub
+#   errors, in order to retry resolution with more flexible constraints when it
+#   fails.
+#
+# * https://github.com/jhawthorn/pub_grub/pull/17, for easier customization of
+#   error messages to include sources and better metadata dependency conflict
+#   explanations.
```

Comment:
Is this goal to get these patches merged and released before bundler is updated or are you planning on carrying these patches?

Reply:
I would like to get these patches merged, but I documented them for now in case review takes a while and we decide to move on. They are mostly cosmetic patches, the one that [more fundamentally changes resolution](https://github.com/jhawthorn/pub_grub/pull/13) has already been merged. So I think we're good even if we decide to move on before these are merged/reviewed upstream, we were also patching Molinillo after all :)

Reply:
All patches are now merged! 🎉